### PR TITLE
Add workflow runner

### PR DESCRIPTION
**Pull request by martmull**
- add workflow runner module
- add an endpoint to trigger a workflow via api
- improve error handling for serverless functions

## Testing
- create 2 serverless functions
- create a workflow
- create this workflow Version
```
{
  "type": "MANUAL",
  "input": {"b": "bb"},
  "nextAction": {
    "name": "step_1",
    "displayName": "Code",
    "type": "CODE",
    "valid": true,
    "settings": {
      "serverlessFunctionId": "Serverless function 1 Id",
      "errorHandlingOptions": {
        "retryOnFailure": {
          "value": false
        },
        "continueOnFailure": {
          "value": false
        }
      }
    },
    "nextAction": {
      "name": "step_1",
      "displayName": "Code",
      "type": "CODE",
      "valid": true,
      "settings": {
        "serverlessFunctionId": "Serverless function 1 Id",
        "errorHandlingOptions": {
          "retryOnFailure": {
            "value": false
          },
          "continueOnFailure": {
            "value": false
          }
        }
      },
      "nextAction": {
        "name": "step_1",
        "displayName": "Code",
        "type": "CODE",
        "valid": true,
        "settings": {
          "serverlessFunctionId": "Serverless function 2 Id",
          "errorHandlingOptions": {
            "retryOnFailure": {
              "value": false
            },
            "continueOnFailure": {
              "value": false
            }
          }
        }
      }
    }
  }
}
```
- call 
```
mutation Trigger {
  triggerWorkflow(workflowVersionId: "WORKFLOW_VERSION_ID") {
    result
  }
}
```
- try when errors are injected in serverless function

### CODE REVIEW

File: packages/twenty-server/src/database/typeorm-seeds/core/feature-flags.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const seedFeatureFlags = async (
       {
         key: FeatureFlagKey.IsWorkflowEnabled,
         workspaceId: workspaceId,
-        value: false,
+        value: true,
```
Comment (thomtrp) [inline packages/twenty-server/src/database/typeorm-seeds/core/feature-flags.ts:61]: should be false by default

File: packages/twenty-server/src/engine/core-modules/workflow/dtos/workflow-trigger-result.dto.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+import { IsObject } from 'class-validator';
+import graphqlTypeJson from 'graphql-type-json';
+
+@ObjectType('WorkflowTriggerResult')
+export class WorkflowTriggerResultDto {
```
Comment (thomtrp) [inline packages/twenty-server/src/engine/core-modules/workflow/dtos/workflow-trigger-result.dto.ts:7]: let's remain consistent with the rest of the code. We usually do not put Dto in the name, or we put DTO

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts [hunk 3]
```diff
@@ -1,13 +1,32 @@
 import { Field, ObjectType } from '@nestjs/graphql';
 
-import { IsObject } from 'class-validator';
+import { IsObject, IsOptional } from 'class-validator';
 import graphqlTypeJson from 'graphql-type-json';
 
 @ObjectType('ServerlessFunctionExecutionResult')
 export class ServerlessFunctionExecutionResultDto {
```
Comment (thomtrp) [inline packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts:7]: same here

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts [hunk 4]
```diff
@@ -1,13 +1,32 @@
...
+    nullable: true,
   })
-  result: JSON;
+  data?: JSON;
+
+  @Field({ description: 'Execution duration in milliseconds' })
+  duration: number;
+
+  @Field({ description: 'Execution status code' })
+  status: 200 | 500;
```
Comment (thomtrp) [inline packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts:19]: not a big fan of having a http status there. I feel we should use an enum for that.

**Review by thomtrp — APPROVED** [review]
🔥

File: packages/twenty-server/src/database/typeorm-seeds/core/feature-flags.ts (hunk 1 above)
Comment (martmull) [reply packages/twenty-server/src/database/typeorm-seeds/core/feature-flags.ts:61]: indeed, commited to fast

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts (hunk 4 above)
Comment (martmull) [reply packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function-execution-result.dto.ts:19]: ok

**Pull request merged**